### PR TITLE
Adds frog hat to donator list and magmu box

### PR DESCRIPTION
**Pull request by VaelophisNyx**
# Document the changes in your pull request

Adds a donator hat for @KillerOrcaCora. They made the sprite

# Spriting

froghat

# Changelog

:cl:  
rscadd: gives Orca Cora a unique donator hat
/:cl:

### CODE REVIEW

**Comment by Mqiib** [comment]
Why is orca getting three donator items

**Comment by VaelophisNyx** [comment]
idfk I just did it as asked

**Comment by cuackles** [comment]
No no, orca doesn't get a third donor item, either add it as a general donor item or as a normal item you can get in-game , if orca wants a more specific reason why, idealy everyone should only have only 1 exclusive doble item cus it's supposed to be special, if you want special clothes the donor items ain't a dresser for you, use the vendors for that, that would be it,  cheers 🥂

File: yogstation/code/modules/clothing/donator.dm [hunk 1]
```diff
@@ -98,6 +98,17 @@
...
+	name = "frog hat"
+	desc = "A strange hat that looks somewhat like a frog?"
+	icon_state = "froghat"
+	item_state = "froghat"
+
+/obj/item/clothing/head/yogs/froghat/Initialize()
+	. = ..()
+	AddComponent(/datum/component/fishingbonus,10)
+
+
```
Comment (JamieD1) [inline yogstation/code/modules/clothing/donator.dm:111]: Remove this space.

**Comment by JamieD1** [comment]
Reopen it and add it to general donator items, yes it can stay in the box.

**Comment by JamieD1** [comment]
Rename this to something less stupid as well.

**Pull request closed without merging**